helm/generic-node/values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+name: "generic-node"
+
+testnetName: "testnet"
+
+image:
+  repository: minaprotocol/mina-daemon
+  pullPolicy: IfNotPresent
+  tag: 1.2.1alpha1-1f98b8b-devnet
+
+features:
+  logsnarkworkgossip:
+    enabled: true
+  enablepeerexchange:
+    enabled: true
+  runtimeConfig:
+    enabled: false
+  maxConnections: 200
+  logTxnPoolGossip:
+    enabled: false
+  exposeGraphql:
+    enabled: false
+  uploadBlocksToGCloud:   # Added this
+    enabled: true
+  customEntrypoint:
+    enabled: false
+    command: ""
+  prometheus:
+    enabled: true
+  persistence:
+    enabled: true
+    name: "generic-node-volume"
+    storageClass: standard-delete
+    mountPath: /data/
+    size: 100Gi
+  privateKeySecret:
+    enabled: true
+    secretName: "generic-node-devnet-bp-key"
+  daemon:
+    args:
```

Comment:
Move to the daemon section so it can be `.Values.daemon.args`, `.Values.rosetta.args`, etc.